script/main.lua

```diff
@@ -1359,7 +1359,7 @@ function (con, args)
 		if args.report == nil then
 			return const.FERR_BAD_SYNTAX
 		end
-		if u.isGlobMod(con) ~= true and u.isAdmin(con) ~= true then
+		if u.isGlobMod(con) ~= true and u.isAdmin(con) ~= true and u.isChanOp ~= true then
```

Comment:
The function "s.isChanOp(con)" exists and should satisfy this check. The adding of it to the connection stuff shouldn't be needed.